src/components/calendar.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MonthsList from '../containers/months-list';
+
+class Calendar extends Component {
+  render() {
+    return (
+      <div className='calendar'>
+        {/* <h2>Calendar</h2> */}
+        <MonthsList />
+      </div>
+    );
+  }
+}
+
+Calendar.propTypes = {
+  //
```

Comment:
(It doesn't need any proptypes yet but instead of removing proptypes i want to remind myself that i intend to use it later ...)